modules/stochastic_tools/include/utils/BootstrapCalculators.h

```diff
@@ -36,22 +48,21 @@ MooseEnum makeBootstrapCalculatorEnum();
  * @param seed Seed for random number generator
  */
 template <typename InType, typename OutType>
-class BootstrapCalculator : public Calculator<InType, std::vector<OutType>>
+class BootstrapCalculator : public libMesh::ParallelObject
 {
 public:
   BootstrapCalculator(const libMesh::ParallelObject & other,
                       const std::string & name,
                       const std::vector<Real> & levels,
                       unsigned int replicates,
                       unsigned int seed,
-                      const StochasticTools::Calculator<InType, OutType> & calc);
+                      StochasticTools::Calculator<InType, OutType> & calc);
```

Comment:
This makes me nervous, computing the Bootstrap statistics should not alter the state of the Calculator. Why is this needed?